index.js

```diff
@@ -17,23 +15,62 @@ function _getBearerToken(mpesa_public_key, mpesa_api_key) {
     return encrypted.toString("base64");
 }
 
+function initialize_api_from_dotenv() {
+    if (!mpesaConfig) {
+        mpesaConfig = {
+            baseUrl: process.env.MPESA_API_HOST,
+            apiKey: process.env.MPESA_API_KEY,
+            publicKey: process.env.MPESA_PUBLIC_KEY,
+            origin: process.env.MPESA_ORIGIN,
+            serviceProviderCode: process.env.MPESA_SERVICE_PROVIDER_CODE
+        };
+        console.log("Using configuration from .env file");
+    } else {
+        console.log("Using the configuration specified in initializeApi()");
+    }
+}
+
+function log_required_config_arg(argName) {
```

Comment:
Might wanna get rid of the "log".

```suggestion
function required_config_arg(argName) {
```